### DISCUSSION TITLE
Translation of assets

### DISCUSSION THREAD

**Discussion by IARI** [WeblateOrg/weblate#6693] [Q&A]
Hi,

We build a mobile application for young children that can not read and hence avoids text except in particular a areas of the app that are specifically designed for adults (the parents). Consequently we use Images, Video and **a lot** of audio material to guide the user.
When it comes to localization of course I would love to have a solution that 
* is deployable as Docker container
* allows product managers and voice actors to do their parts of the process with a clean webbased ui
* allows developers to tightly integrate with the CI process.

I'm completely new to weblate and have not yet ever set up an instance yet.

My Research has not yet given many useful results - however it seems like weblate is really one of if not the biggest localization platform which appears to have powerful features.
However I can not find anything with respect to the localization of Assets that are not plain text (Audio, Video, Images)

So my question is:
- Am I missing something or is nothing like that supported in weblate?
- Is there even anything out there that does support Assets?

**Comment by rffontenelle** [comment]
While not replying the question directly, I have one suggestion: the now-archived "gnome-getting-started-docs" was a documentation for GNOME Desktop containing animations and SVG images.

I mentioned this because I translated Gettext PO files of strings from both type of assets which would be generated as translated versions on build time. 

Maybe you can learn something from this and make those strings translatable in Weblate in the same way?

Sadly I don't know exactly how it works, but I bet you could find devs involved and ask questions.

Here is the source code repository: https://gitlab.gnome.org/Archive/gnome-getting-started-docs
